spec/controllers/authors_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe AuthorsController, type: :controller do
+  describe 'POST /authors' do
+    context 'with valid params' do
+      before do
+        post :create,
+             params: {
+               first_name: 'Bill',
+               last_name: 'Gates',
+               email: 'gatess@ll.cc'
+             }
+      end
+
+      it { should render_template(:new) }
```

Comment:
Each of these 3 examples will run separate post call. Consider optimizing tests for all possible minimums.